lib/errors-middleware.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const createError = require('http-errors');
+const debug = require('debug')('people:errors-middleware');
+
+module.exports = function(err, req, res, next) {
+  if(err.status) {
+    debug('user error');
+
+    res.status(err.status).send(err.name);
+    next();
+    return;
+  }
+
```

Comment:
You need to fill out your error middleware a bit to handle all of your errors. For instance: when you pass invalid or missing values in a post request mongo is going to kick back and error called 'ValidationError' but it doesn't assign a status to that error so when your function enters the error middleware you don't have a function to deal with that so it just runs a 500 error. This is why your tests were failing. 

Alternatively you want to catch that error with a conditional and assign it a status. Example:

``` javascript
if(err.name === 'ValidationError') {
    debug('Validation Error');

    err = createError(400, err.message);
    res.status(err.status).send(err.name);
    next();
    return;
  }
```